recipes/diligentgraphics-spirv-tools/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(SPIRV-Tools REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME}_c test_package.c)
+if(TARGET SPIRV-Tools-shared)
+    target_link_libraries(${PROJECT_NAME}_c SPIRV-Tools-shared)
+elseif(TARGET SPIRV-Tools-static)
+    target_link_libraries(${PROJECT_NAME}_c SPIRV-Tools-static)
+else()
+    if(NOT TARGET SPIRV-Tools)
+        message(FATAL_ERROR "There is no SPIRV-Tools target")
+    endif()
+    target_link_libraries(${PROJECT_NAME}_c SPIRV-Tools)
```

I think you can throw an error unconditionally?
```suggestion
    message(FATAL_ERROR "Neither target SPIRV-Tools-shared or target SPIRV-Tools-shared exist")
```